docs/split-inputs.md

```diff
@@ -0,0 +1,140 @@
+# How To Split A Fuzzer-Generated Input Into Several
+
+This document describes several recommended strategies for splitting
+a single fuzzer-generated input into several.
+
+# Examples
+Splitting a fuzzer-generated input into several independent inputs is required
+very often. Some examples:
+
+* Fuzzing a regular expression library requires
+   * The regular expression (RE)
+   * Flags for RE compilation and matching
+   * A string to search the RE in
+
+* Fuzzing an audio/video format decoder often requires
+  * Decoding flags
+  * Several frames
+
+* Fuzzing a XSLT or CSS library requires
+  * The stylesheet input
+  * The XML/HTML input
+
+* Fuzzing a font-rendering library requires
+  * The font file
+  * The text to render
+  * The rendering flags
+
+* Fuzzing a database library may require
+  * The query text
+  * The database state
+
+TODO: more examples?
+
+# Common Data Format
+
+When trying to split the fuzzer-generated input into several,
+the first question one needs to ask is whether the input format is common,
+i.e. is it used or processed by other libraries, APIs, of fuzz targets.
+
+If the data format is common (e.g. a widely used media format or network packet
+format) then it is highly desirable for a fuzz target to consume exactly this
+data format, and not some custom modification.
+This way it will be easier to procure a seed corpus for this fuzz target
```

Comment:
It doesn't feel super clear here that we propose procuring a good seed corpus "instead of manually splitting the data and trying to reconstruct the format inside the fuzz target" -- I'd add something like that, just to explicitly say that we recommend against splitting the input in such cases.

Reply:
Mmm.. Looks clear to me. Maybe you could propose a better wording as a separate PR?

Reply:
Sounds good!